lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -189,6 +199,12 @@ def _configure_base(module, base, conf_file, disable_gpg_check):
     # Don't prompt for user confirmations
     conf.assumeyes = True
 
+    # Set installroot
+    if installroot and installroot != '/':
```

Comment:
Should the default in `argument_spec` (and the docs) just be set to `/`? then this can be simplified?